biscuit-auth/src/datalog/mod.rs

```diff
@@ -737,6 +743,8 @@ pub struct RunLimits {
     pub max_iterations: u64,
     /// maximum execution time
     pub max_time: Duration,
+
+    pub extern_funcs: HashMap<String, ExternFunc>,
```

Comment:
I am not sure putting the functions in `RunLimits` is the right way. I'd prefer if it was stored at the `World` level, so we can make sure they are always applied